lib/POMDPTools/src/Testing/model.jl

```diff
@@ -1,25 +1,28 @@
 """
-    has_consistent_distributions(m::MDP)
-    has_consistent_distributions(m::POMDP)
+    has_consistent_distributions(m::MDP; [atol=0])
+    has_consistent_distributions(m::POMDP; [atol=0)
```

Comment:
missing closing bracket?

Reply:
thanks for catching! I actually looked at some Base docstrings and they don't have brackets in this case, so I completely left them out.